src/reference/config/writing-platform-configs.rst

```diff
@@ -13,16 +13,21 @@ Writing Platform Configurations
    - :cylc:conf:`global.cylc[platforms]`
    - :cylc:conf:`global.cylc[platforms][<platform name>]install target`
 
+.. _ListingAvailablePlatforms:
+
+Listing available platforms
+---------------------------
+
 If you are working on an institutional network platforms may already
 have been configured for you.
 
-.. TODO update the command below after implementing a platform
-   listing command.
+To see a list of available platforms::
+
+   cylc config --platform-names
 
-To see available platforms::
+To see the full configuration of available platforms::
 
-   cylc config -i [platforms]
-   cylc config -i [platform groups]
+   cylc config --platform-meta
```

Comment:
Note https://github.com/cylc/cylc-flow/pull/4576#discussion_r791218359

(Change to `--platforms`?).

And it wouldn't hurt to mention that this option is just shorthand for `cylc config -i 'platforms' -i 'platform groups'`.

Reply:
```suggestion
   cylc config --platform-meta
   
This is equivalent to `cylc config -i 'platforms' -i 'platform groups'`
```

Reply:
@hjoliver  I have added a comment about platform meta being the same as `cylc config -i 'platforms' -i 'platform groups'`. I deliberately avoided using `--platforms` because I wasn't sure which of the above it should map to.

Reply:
(My point was, in the cylc-flow PR, `--platform-meta` is an incorrect name because it prints the full platforms config, not just the metadata).

Reply:
Ok, Reasonable